thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyMergeExecutor.java

```diff
@@ -74,21 +75,48 @@ public void updateMergedResults() {
               "Merge strategy " + mergeConfig.getMergeStrategy() + " not supported");
         }
       }
+      output.forEach(this::updateMergedScoreAndPersist);
     }
   }
 
-  void performMergeBasedOnFunctionId(AnomalyFunctionSpec function,
+  private void updateMergedScoreAndPersist(AnomalyMergedResult mergedResult) {
```

Comment:
Do we have the method to update severity of merged anomaly? How is that done? To do that, we need to get the min and max timestamp of the anomalies and repull the time series data to calculate baseline and current observation to get lift or difference.